fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1873,21 +1873,21 @@ stop_routine_load_stmt ::=
     ;
 
 show_routine_load_stmt ::=
-    KW_SHOW KW_ROUTINE KW_LOAD KW_FOR job_label:jobLabel
+    KW_SHOW KW_ROUTINE KW_LOAD KW_FOR job_label:jobLabel opt_wild_where
```

Comment:
This means 'show routine load for xxx like xxx'.
But I understand what you mean should be for and like pick only one?

Reply:
my fault, I will fix it.